pom.xml

```diff
@@ -23,7 +23,7 @@
         <module>spring-petclinic-config-server</module>
         <module>spring-petclinic-discovery-server</module>
         <module>spring-petclinic-api-gateway</module>
-        <!-- <module>spring-petclinic-messaging-emulator</module> -->
+        <module>spring-petclinic-messaging-emulator</module>
```

Comment:
keep this one commented out please in the labstarter. The lab starts without building the emulator and then builds it separately later in the lab.